connection/database.feature

```diff
@@ -11,9 +11,16 @@ Feature: Connection Database
     Given connection is open: true
     Given connection has 0 databases
 
-  Scenario: create one database
-    When connection create database: alice
-    Then connection has database: alice
+  Scenario Outline: create one database with name <name>
+    When connection create database: <name>
+    Then connection has database: <name>
+    Examples:
+      | name                                                                  |
+      | alice                                                                 |
+      | ALICE                                                                 |
+      | cAn-be_Like-that_WITH-a_pretty-looooooooooooong_name-and·even‿a·smile |
+      | 資料庫                                                                   |
```

Comment:
Auto-formatting does not understand Chinese symbols well in terms of the space they take, so the tables will be like this.